### PR TITLE
Implement some IAccountServiceForApplication functions

### PR DESCRIPTION
**Pull request by MK73DS**
This PR stubs some functions in [IAccountServiceForApplication](https://switchbrew.org/wiki/Account_services#acc:u0) :
- IAccountServiceForApplication::GetUserCount (0x0)
- IAccountServiceForApplication::InitializeApplicationInfo (0x8C)
- IAccountServiceForApplication::IsUserAccountSwitchLocked (0x96)

This is required by New Pokémon Snap, implementing these services makes this game boot a bit further and display a (broken) splash screen.

image

### CODE REVIEW

**Comment by maytinhdibo** [comment]
great

**Review by PixelyIon — CHANGES_REQUESTED** [review]
Good job with this, you got all the basics of implementation right and most comments concern the specifics of comments and other minor issues.

File: app/src/main/cpp/skyline/services/account/IAccountServiceForApplication.cpp [hunk 1]
```diff
@@ -76,4 +76,20 @@ namespace skyline::service::account {
         manager.RegisterService(SRVREG(IManagerForApplication), session, response);
         return {};
     }
+
+    Result IAccountServiceForApplication::InitializeApplicationInfo(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        auto pid{request.Pop<u64>()};
+        u64 pid_placeholder{request.Pop<u64>()};
```
[the remark this answers is not in the document]
Comment (bylaws) [reply app/src/main/cpp/skyline/services/account/IAccountServiceForApplication.cpp:82]: Don't do that since we could need them

**Review by PixelyIon — APPROVED** [review]
LGTM!

**Comment by maytinhdibo** [comment]
[FILE_20220310_125347_emulation.sklog.txt](https://github.com/skyline-emu/skyline/files/8220754/FILE_20220310_125347_emulation.sklog.txt)
Pokemon Snap still crash

**Comment by PixelyIon** [comment]
@sauradipdebnath That's not how the code works, I understand you want to help but this just clogs up the repository. I will be doing a temporary ban on the next unrelated comment.

**Pull request merged**